### PR TITLE
Remove args and working directory for Source column

### LINKED ISSUE #1820: Push more details for executables in the details view and out of the grid
Working directory and the args should probably be moved out of the grid view.

image

### PR DESCRIPTION
**Pull request by tlmii**
Resolves #1820 

Removes args and working directory from the source column of the resources grid for executables. Both are already in the details view, so nothing to add there (arguably we might want to change the way we display args in the details pane - that can be a separate change though). 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/1842)

### CODE REVIEW

**Review by adamint — APPROVED** [review]

**Comment by JamesNK** [comment]
I agree the working directory should be removed. But args seem important. Are they a problem if they're trimmed with an ellipsis if there isn't space available?

**Comment by tlmii** [comment]
> I agree the working directory should be removed. But args seem important.
> 
> 
> 
> Displaying exe name plus args and then trimming away overflow seems fine. The tooltip can be exe name plus args.

I tend to agree. @davidfowl thoughts since you filed the issue?

**Comment by davidfowl** [comment]
Seems like a fine idea.

**Comment by tlmii** [comment]
Took me a couple attempts to remove some unrelated changes to the .resx file. Not sure what my toolset is doing differently from others. Regardless, this change should be done now.

**Review by adamint — APPROVED** [review]

**Review by kvenkatrajan — APPROVED** [review]

**Comment by davidfowl** [comment]
Thanks @tlmii !

File: src/Aspire.Dashboard/Components/ResourcesGridColumns/SourceColumnDisplay.razor [hunk 1]
```diff
@@ -13,7 +13,7 @@
                PreCopyToolTip="@Loc[nameof(Columns.SourceColumnSourceCopyFullPathToClipboard)]"
                ToolTip="@projectPath" />
 }
-else if (Resource.TryGetExecutablePath(out var executablePath) && Resource.TryGetWorkingDirectory(out var workingDirectory))
```
Comment (drewnoakes) [inline src/Aspire.Dashboard/Components/ResourcesGridColumns/SourceColumnDisplay.razor:16]: Can we now remove the `TryGetWorkingDirectory` method? I doubt it's used anywhere else.

Comment (drewnoakes) [reply src/Aspire.Dashboard/Components/ResourcesGridColumns/SourceColumnDisplay.razor:16]: https://github.com/dotnet/aspire/pull/1856

**Pull request merged**